lib/gtfsplus/actions/gtfsplus.js

```diff
@@ -173,107 +175,104 @@ export function uploadGtfsPlusFeed (feedVersionId: string, file: File) {
       return
     }
     return uploadFile({file, url, token: user.token})
-      .then(result => {
-        console.log(result)
-        return dispatch(uploadedGtfsPlusFeed())
-      })
+      .then(result => dispatch(uploadedGtfsPlusFeed()))
   }
 }
 
+type GTFSPlusEntity = {
+  rowIndex: number,
+  origRowIndex: number,
+  stop_id?: string,
+  route_id?: string,
+  [string]: any
```

Comment:
This `[string]: any` is way too broad and allows for anything.  Please replace this with something more specific.